docs/user/installation.rst

```diff
@@ -20,16 +20,13 @@ Install from PyPI
 .. note:: Lasagne hasn't been released as of now.  The only way to
           install it for now is from source.
 
-Simply run:
+To install release 1.0 of Lasagne from PyPI, run the following
+commands:
 
 .. code-block:: bash
 
-  pip install Lasagne
-
-It's also a good practice to install dependencies with exactly the
-same version numbers that the release was made with.  You can find the
-``requirements.txt`` that defines those version numbers in the top
-level directory of the Lasagne source tree.
+  pip install -r https://raw.githubusercontent.com/Lasagne/Lasagne/1.0/requirements.txt
+  pip install Lasagne==1.0
```

Comment:
Just to avoid possible confusion, I'd suggest to use `0.1` as the example and update it whenever we do a release. Otherwise I bet we'll soon have github issues / mails about `pip install Lasagne==1.0` not working :)